app/events/[slug]/page.tsx

```diff
@@ -1,16 +1,181 @@
+  const hasSponsors: boolean =
+    Array.isArray(orgEvent.sponsor) && orgEvent.sponsor.length > 0;
+  const hasEventAssets: boolean =
+    Array.isArray(orgEvent.eventAsset) && orgEvent.sponsor.length > 0;
+
   return (
     <main>
-      <h1>Event = {orgEvent.eventName}</h1>
+      <div className="flex flex-col-reverse md:flex-row" id="main-event">
+        <div className="mx-10" id="main-event-info">
+          <h1 className="my-6 text-2xl font-bold">{orgEvent.eventName}</h1>
+          {renderRichTextToReactComponent(
+            orgEvent.eventSummary as unknown as Document,
+            eventSummaryClassNames,
+          )}
+          <p>Event Date: {formattedDateTime}</p>
+          <div className="flex flex-col">
+            <button className="my-5 w-44 rounded-md bg-[#0076AD] py-1 text-lg text-white">
+              ATTEND EVENT
+            </button>
+            <button className="mb-8 text-left underline">DIRECTIONS</button>
+          </div>
+        </div>
+        <div id="main-event-asset">
+          <Image
+            alt="Main Event Photo"
+            src={eventPhoto.url}
+            width={eventPhoto.details.image.width}
+            height={eventPhoto.details.image.height}
+            className="h-64 object-cover"
+          />
+        </div>
+      </div>
+      <h2 className="mx-6 mb-4 text-lg font-semibold text-[#32B8DE]">
+        PATIENT AMBASSADORS
+      </h2>
+      <div className="mx-6 grid grid-cols-1 place-items-center md:mx-auto md:flex md:justify-center md:gap-8">
+        {hasPatientAmbassadors &&
+          orgEvent.patientAmbassador.map(
+            (patientObject: { fields: MinimalCardType }) => (
+              <MinimalCard
+                key={patientObject.fields.title}
+                cardContent={patientObject.fields}
+                styleProps={patientMinimalCardStyleProps}
+              />
+            ),
+          )}
+      </div>
+      <div id="event-details" className="flex flex-col-reverse">
+        <p className="mx-6 mb-6">{orgEvent.eventDetails}</p>
+        <DynamicImage
+          alt="Event Photo"
+          src={eventDetails?.fields?.file.url}
+          width={eventDetails?.fields?.file.details.image.width}
+          height={eventDetails?.fields?.file.details.image.height}
+          className="my-8 h-52 object-cover"
+        />
+      </div>
+      <div id="event-assets" className="flex">
+        {hasEventAssets &&
+          orgEvent.eventAsset.map((asset: NestedAssetType) => (
+            <DynamicImage
+              alt="event-assets"
+              key={asset.sys.id}
+              src={asset.fields.file.url}
+              width={asset.fields.file.details.image.width}
+              height={asset.fields.file.details.image.height}
+              className="mb-8"
+            />
+          ))}
+      </div>
+      {/* change date according to event year */}
+      <p className="ml-36 w-44 text-lg font-semibold">
+        THANK YOU TO OUR 2023 SPONSORS
```

Comment:
Maybe we could make this a field in Contentful? We can ask @jtorreggiani about this